src/controllers/item.js

```diff
@@ -1,6 +1,13 @@
+const ItemModel = require('../models/item');
+
 module.exports = {
-  getAllItems: (req, res) => {
-    res.send('All items');
+  getAllItems: async (req, res) => {
+    try {
+      const items = await ItemModel.find();
+      res.json(items);
+    } catch (err) {
+      res.status(422).json({ message: err.message });
+    }
```

Comment:
After Shrreya reviews the other PR and approves it and it gets merged into main don't forget to pull main and push it to this branch because these changes are showing on both PRs